Program.cs

```diff
@@ -98,7 +98,26 @@ static void Main(string[] args)
                     {
                         Console.WriteLine("(response not recognized)");
                     }
-                } while (intersectionChoice.ToLower() != "left"); 
+                } while (intersectionChoice.ToLower() != "left");
+
+                string response2 = GetUserResponse("Do you want to follow him?");
+                if (response2.ToLower() == "yes")
+                {
+                    Console.WriteLine("You follow him from a distance while trying to remain discrete." +
+                    	"As you get closer you see him enter a building.");
+                   
+                     string response3 = GetUserResponse("Do you want to risk it and enter after him?");
```

Comment:
This line isn't aligned with the others around it. Consider removing space on the left so that it's in line